README.md

```diff
@@ -11,10 +11,20 @@ This repository translates our design system to code and publishes the correspon
 
 Run `npm run storybook` from the root directory to start storybook locally.
 
+### Deploying to GitHub Pages
+
+This should happen after every publish so that our Storybook documentation accurately reflects the most recent consumable versions of our components.
```

Comment:
Is there a way we can make this happen automatically after every publish?

Reply:
there are probably a lot of ways to do this, but 2 i was thinking:
- there is a `postpublish` script that could map to a job that automatically does this, but it will run even if the publish fails, so i don't really like this option
- build a github action that listens for tags to be pushed to a certain branch and automatically deploys storybook. every time packages are published, the publish commit is tagged, so this should only happen after successful publishes. i prefer this option, but have some other work in-flight so i logged it as [DSCO-43](https://codedotorg.atlassian.net/browse/DSCO-43) to revisit